plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/FunnyCommandsConfiguration.java

```diff
@@ -151,7 +150,7 @@ private CommandComponents(String group) {
             this.group = group;
         }
 
-        private CommandComponents command(String name, FunnyCommand configuration, Class<?> command) {
+        private CommandComponents command(String name, net.dzikoysk.funnyguilds.config.sections.CommandsConfiguration.FunnyCommand configuration, Class<?> command) {
```

Comment:
Czemu tu jest full ścieżka, jak masz konflikt nazw to po prostu go popraw, chyba że to typo

Reply:
Typo